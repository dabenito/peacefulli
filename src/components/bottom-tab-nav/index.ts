export { Nav } from "./bottom-tab-nav";
