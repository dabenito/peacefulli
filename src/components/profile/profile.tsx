import React, { PureComponent } from "react";
import { View, Text } from "react-native";
import { styles } from "./style";

export interface ProfileProps {}

export interface ProfileState {}

export class Profile extends PureComponent<ProfileProps, ProfileState> {
  constructor(props: ProfileProps) {
    super(props);
  }

  render() {
    return (
      <View style={styles.container}>
        <Text>TODO: Create Profile View!</Text>
      </View>
    );
  }
}
