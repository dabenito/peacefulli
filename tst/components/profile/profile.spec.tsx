import React from "react";
import { render, cleanup } from "@testing-library/react-native";
import { Profile } from "../../../src/components/profile";

describe("Profile Tests", () => {
  afterEach(cleanup);

  // TODO: update this file when implementing profile page
  it("loading profile page", async () => {
    const profile = render(<Profile />);
    expect(profile).toMatchSnapshot();
  });
});
