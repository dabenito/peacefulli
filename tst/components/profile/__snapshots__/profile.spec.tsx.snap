// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile Tests loading profile page 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    TODO: Create Profile View!
  </Text>
</View>
`;
